ssv/aggregator.go

```diff
@@ -14,10 +14,11 @@ import (
 type AggregatorRunner struct {
 	BaseRunner *BaseRunner
 
-	beacon   BeaconNode
-	network  Network
-	signer   types.KeyManager
-	valCheck qbft.ProposedValueCheckF
+	beacon         BeaconNode
+	network        Network
+	signer         types.KeyManager
```

Comment:
if I understand correctly at some other places you call this `types.SSVShareSigner`, it's best to decide on similar name/type. unless you create local interfaces for each struct with only the subset of functions they need.

Reply:
The `KeyManager` includes the `SSVShareSigner`. Do you mean renaming the attribute `signer`?

Reply:
Oh I just saw SSVShareSigner is part of keymanager. sorry. I thought you splitted them somehow. resolving.